src/Uintra20/Content/uintra20/src/app/feature/project/specific/groups/groups-form/groups-form.component.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { TITLE_MAX_LENGTH } from 'src/app/constants/activity/create/activity-create-const';
+import { GroupsService } from 'src/app/feature/project/specific/groups/groups.service';
+import { finalize } from 'rxjs/operators';
+import { MAX_FILES_FOR_SINGLE } from 'src/app/constants/dropzone/drop-zone.const';
+
+@Component({
+  selector: 'groups-form',
+  templateUrl: './groups-form.component.html',
+  styleUrls: ['./groups-form.component.less']
+})
+export class GroupsFormComponent {
+  @Input() data: any;
+  @Input('edit') edit: boolean;
```

Comment:
try to use attribute
 [https://angular.io/api/core/Attribute](https://angular.io/api/core/Attribute) 
[https://netbasal.com/getting-to-know-the-attribute-decorator-in-angular-4f7c9fb61243](https://netbasal.com/getting-to-know-the-attribute-decorator-in-angular-4f7c9fb61243)